### PR TITLE
enh: remove arbitrum dai

### PR DESCRIPTION
**Pull request by bard**
closes: #286

### CODE REVIEW

**Review by vacekj — APPROVED** [review]

**Review by vacekj — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -274,9 +274,7 @@ async function catchupAndWatchChain(
       {
         toBlock: config.toBlock,
         logger: indexerLogger,
-        eventCacheDirectory: config.cacheDir
-          ? path.join(config.cacheDir, "events")
-          : null,
+        eventCacheDirectory: null,
```
Comment (boudra) [inline src/index.ts:277]: this change seems to have been checked in by accident? but it's the same in main so should be ok to merge

**Review by boudra — DISMISSED** [review]

**Review by boudra — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (bard) [reply src/index.ts:277]: Yes, it's unrelated to the change, I don't remember how it made its way here.

**Review by boudra — DISMISSED** [review]

**Comment by boudra** [comment]
Something really weird is happening here, everytime I approve the branch Github dismisses it automatically 🤔

**Review by boudra — APPROVED** [review]

**Comment by bard** [comment]
Not only that, it also blames it on me:

image

I guess it's due to the merge base, which is probably why also that extra line was there.

**Pull request merged**